dev-modules/babel-plugin-probe-gl/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "babel-plugin-probe-gl",
```

Comment:
On the topic of naming,
- Do we expect to have only one plugin for probe, possibly doing multiple things in the future, or do we want to qualify the name with what the plugin does, e.g. `babel-plugin-probe.gl-strip-logs`?
- I expect it is not worth the hassle to use the probe.gl namespace on a babel plugin `@probe.gl/babel-plugin-strip-logs`)

Reply:
As for extensions to this, I am interested in having a `babel-plugin-strip-assert`:
- The existing npm module for this does not seem to work or be maintained. I expect a small variant of this plugin could work for that case.
- We could just make one plugin `babel-plugin-strip-functions` that handles both functions and methods. 
- The main weirdness is that generic plugin would come with probe.gl specific defaults, but we can always call it `babel-plugin-probe.gl-strip-functions` to get around that.